cogeo_mosaic/backends/dynamodb.py

```diff
@@ -183,8 +183,8 @@ def _create_table(self, billing_mode: str = "PAY_PER_REQUEST", **kwargs: Any):
             {"AttributeName": "quadkey", "AttributeType": "S"},
         ]
         key_schema = [
-            {"AttributeName": "mosaicId", "KeyType": "RANGE"},
-            {"AttributeName": "quadkey", "KeyType": "HASH"},
+            {"AttributeName": "mosaicId", "KeyType": "HASH"},
+            {"AttributeName": "quadkey", "KeyType": "RANGE"},
```

Comment:
fyi quadkey is currently a hash key I think

Reply:
@kylebarron do you think this change is a problem ?

Reply:
No actually this change makes sense, the hash is the partitioning and the range is a sorted index within the partition. Thus it makes sense for Mosaic id to be hash and the other range. I don't think both can be hash? https://stackoverflow.com/questions/27329461/what-is-hash-and-range-primary-key

Reply:
> I don't think both can be hash
Confirming that both cannot be hash key. DynamoDB records are identified by a unique primary key, which can be composed of just a hash key (which was originally the case, with `quadKey` as the hash key) or a hash + range key (which is now the case, with `mosaic_id` as hash key and `quadKey` as range key)